indexers/general-squid/src/blocks/databaseOperations.ts

```diff
@@ -40,21 +44,23 @@ export async function saveBlock(
     spacePledged,
     blockchainSize: historySize,
     author,
-    extrinsicsCount: 0,
-    callsCount: 0,
-    eventsCount: 0,
-    logsCount: 0,
+    extrinsicsCount: extrinsicsCount,
+    callsCount: callsCount,
+    eventsCount: eventsCount,
+    logsCount: logsCount,
```

Comment:
this can be simplify
```suggestion
    extrinsicsCount,
    callsCount,
    eventsCount,
    logsCount,
```